mlflow/store/tracking/file_store.py

```diff
@@ -221,13 +222,18 @@ def _get_active_experiments(self, full_path=False):
     def _get_deleted_experiments(self, full_path=False):
         return list_subdirs(self.trash_folder, full_path)
 
-    def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
+    def list_experiments(self, view_type=ViewType.ACTIVE_ONLY, max_results=None, page_token=None):
```

Comment:
Per Slack discussion, if we remove max_results default value in the proto field, the API should always provide a default value instead of None. For example:

def list_experiments(self, view_type=ViewType.ACTIVE_ONLY, max_results=1000, page_token=None):
  ...

ditto for the other API list_experiments(...) below.

Reply:
Added the defaulut values!